Add show lacp detail and statistics commands

whatupmiked opened the pull request:
```Signed-off-by: Michael Doyle <dev@whatupmiked.com>```

nickbroon commented:
Looks like it now needs a rebase.

3chas3 reviewed (changes requested):
The configuration state should be modeled in YANG. The "show" commands then retrieve this state and format it for display. We do this to play nicely with other Netconf implementations. I don't believe there is an externally available guide on how to do this. However, there are other examples of this in other DANOS repositories. vyatta-poe is a fairly trivial example.

The YANG should be extended to include a "config false;" state tree. A get-state node should be provided that emits JSON that corresponds to the state tree. The show commands simply retrieve this tree and format the results for the end user.

Please license these files with a compatible open source license.

whatupmiked committed:
Add show lacp detail and statistics command through configd

Signed-off-by: Michael Doyle <dev@whatupmiked.com>

whatupmiked commented:
Got a chance to work on this last weekend and I believe I've incorporated the changes you were looking for and _I think_ it's ready to go.

Wasn't quite sure what to put for the licensing so I just added GPLv2. Added author because I wasn't quite sure how to go about adding a license statement without an author, can remove that if that's what you are looking for.  know very little about licensing. I had thought DCO was enough but if not, or if there is a specific way to license these I think https://danosproject.atlassian.net/wiki/spaces/DAN/pages/3276830/Contributing+to+the+DANOS+source+code should also be updated.
